server/services/interventionsService.test.ts

```diff
@@ -434,6 +434,41 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(referral.furtherInformation).toBe('Some information about the service user')
     })
 
+    it('returns the updated referral when selecting the relevant sentence', async () => {
+      await provider.addInteraction({
+        state: 'There is an existing draft referral with ID of d496e4a7-7cc1-44ea-ba67-c295084f1962',
+        uponReceiving: 'a PATCH request to update the sentence ID',
+        withRequest: {
+          method: 'PATCH',
+          path: '/draft-referral/d496e4a7-7cc1-44ea-ba67-c295084f1962',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token}`,
+          },
+          body: {
+            sentenceId: 2600295124,
+          },
+        },
+        willRespondWith: {
+          status: 200,
+          body: {
+            id: 'd496e4a7-7cc1-44ea-ba67-c295084f1962',
+            sentenceId: 2600295124,
```

Comment:
Could we call this `relevantSentenceId` or `contextSentenceId`? To encode that we select the sentence to set the referral in context?

Reply:
How about `selectedSentenceId`?

Reply:
`relevantSentenceId` sounds good to me